workshop.ipynb

```diff
@@ -9,8 +9,9 @@
     "In this workshop we will learn the basic functionalities of Git and how these can improve our team workflow.\n",
     "Most of this workshop will be conducted using a bash shell and as such some basic command line usage will also be exposed\n",
```

Comment:
"will also be exposed" doesn't sound good, I suggest "will also be **shown and explained**"